local/compose/containers.go

```diff
@@ -70,3 +70,9 @@ func (containers Containers) names() []string {
 	}
 	return names
 }
+
+func (containers Containers) forEach(fn func(moby.Container)) {
```

Comment:
forEach in go... don't push things too far :)

Reply:
(｡◕‿◕｡)